config.schema.json

```diff
@@ -7,7 +7,8 @@
       "type": [
         "string",
         "null"
-      ]
+      ],
+      "format": "regex"
```

Comment:
This file is code-generated.  You'll need to specify this in https://github.com/crate-ci/committed/blob/master/crates/committed/src/config.rs using `schemars`

Reply:
Done.